Elsewhen/ElsewhenApp.swift

```diff
@@ -26,7 +26,8 @@ struct ElsewhenApp: App {
         WindowGroup {
             #if os(macOS)
             MacContentView()
-                .frame(minWidth: 700, minHeight: 400)
+                .frame(minWidth: 400, minHeight: 500)
```

Comment:
I'm assuming you tested the minimum width and height is enough? :)

Reply:
I did!